code/API_definitions/qod-api.yaml

```
@@ -73,14 +73,11 @@ externalDocs:
 security:
   - oAuth2ClientCredentials: []
 servers:
-  - url: "{apiRoot}/{basePath}"
+  - url: "{apiRoot}/qod/v0"
     variables:
       apiRoot:
         default: http://localhost:9091
         description: API root
```

Within last QoD call there was a request to clarify that apiRoot could be also include a path after the host. A proposal for that:

```suggestion
        description: API root, defined by the service provider, e.g. `api.example.com` or `api.example.com/somepath`
```